### PR TITLE
updating sdk to add DELETE /sessions requests for Auth0

### PR DESCRIPTION
**Pull request by kevin-wu24**
Adding functions to hit the `DELETE /sessions` endpoint for SSO and normal login.

### CODE REVIEW

File: auth.go [hunk 1]
```diff
@@ -56,6 +56,25 @@ func (a *AuthService) login(path string, req *AuthenticateRequest) (*Authenticat
 	return res, nil
 }
 
+// Logout attempts to log a user in with an Auth0 ID token, returning either a (CCloud) token or an error.
```
Comment (brianstrauch) [inline auth.go:59]: Looks like this text was copied?

File: auth.go [hunk 2]
```diff
@@ -56,6 +56,25 @@ func (a *AuthService) login(path string, req *AuthenticateRequest) (*Authenticat
...
+func (a *AuthService) Logout(req *AuthenticateRequest) (*AuthenticateReply, error) {
+	return a.logout("/api/sessions", req)
+}
+
+func (a *AuthService) OktaLogout(req *AuthenticateRequest) (*AuthenticateReply, error) {
```
Comment (brianstrauch) [inline auth.go:64]: Is this for SSO?

Comment (kevin-wu24) [reply auth.go:64]: Yeah, it looks like there's also a route for SSO: https://github.com/confluentinc/cc-gateway-service/blob/0b88d5d9b0015a0d7e91d6869dab330eeb055125/http/proxy_cloudv1_routes.go#L146

**Review by brianstrauch — APPROVED** [review]
+1, as long as you have tested that this code can be successfully called from the CLI

**Pull request merged**